models/database_models.py

```
@@ -1,7 +1,8 @@
 """Database models using SQLAlchemy."""
 
-from sqlalchemy import Column, String, DateTime, Boolean, BigInteger, Float, Text, Enum
+from sqlalchemy import Column, String, DateTime, Boolean, BigInteger, Float, Text, Enum, ForeignKey
```

Ruff (0.12.2)

`sqlalchemy.BigInteger` imported but unused

Remove unused import

(F401)